### PR TITLE
Shadow-Fix

### PR DESCRIPTION
**Pull request by afraidofdark**
RenderJob's wasn't using skinned characters' aabb override,
Due to that, initial mesh bb was in use either uninitialized or too big which breaks the shadow frustum calculation
This is fixed. Render Job's uses aabb over the characters.

Sometimes Level designers import other scenes and left objects far away from the scene which enlarges the shadow volume too much. Statistically outliers found and removed.

In short,
Outliers and entities with Invalid boundary found and removed.

To do this a command added "CheckSceneHealt" in the future, we will add one more command for assets which will check if the assets have old deprecated data and fixes them.

### CODE REVIEW

**Review by benanil — APPROVED** [review]
didn't understand what BoolCheck does but other than that everything looks ok.

**Comment by afraidofdark** [comment]
> BoolCheck

BoolCheck controls if the commands ends with { nothing, 0, 1} and if nothing or 0 provided with the command return value is false, otherwise true.
You can use this info to proceed with your command.
